CMakePresets.json

```diff
@@ -0,0 +1,287 @@
+{
+    "version": 6,
+    "cmakeMinimumRequired": {
+      "major": 3,
+      "minor": 14,
+      "patch": 0
+    },
+    "configurePresets": [
+      {
+        "name": "common",
+        "hidden": true,
+        "cacheVariables": {
+          "FREECAD_USE_PYBIND11": {
+            "type": "BOOL",
+            "value": "ON"
+          }
+        }
+      },
+      {
+        "name": "debug",
+        "displayName": "Debug",
+        "description": "Default debug profile",
+        "binaryDir": "${sourceDir}/build/debug",
+        "inherits": [
+          "common"
+        ],
+        "cacheVariables": {
+          "CMAKE_BUILD_TYPE": {
+            "type": "STRING",
+            "value": "Debug"
+          }
+        }
+      },
+      {
+        "name": "release",
+        "displayName": "Release",
+        "description": "Default release profile",
+        "binaryDir": "${sourceDir}/build/release",
+        "inherits": [
+          "common"
+        ],
+        "cacheVariables": {
+          "CMAKE_BUILD_TYPE": {
+            "type": "STRING",
+            "value": "Release"
+          }
+        }
+      },
+      {
+        "name": "conda",
+        "hidden": true,
+        "generator": "Ninja",
+        "cacheVariables": {
+          "Boost_NO_BOOST_CMAKE": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_DYNAMIC_LINK_PYTHON": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_FEM_NETGEN": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "BUILD_WITH_CONDA": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "ENABLE_DEVELOPER_TESTS": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "FREECAD_LIBPACK_USE": {
+            "type": "BOOL",
+            "value": "OFF"
+          },
+          "FREECAD_USE_EXTERNAL_FMT": {
+            "type": "BOOL",
+            "value": "ON"
+          },
+          "FREECAD_USE_EXTERNAL_SMESH": {
+            "type": "BOOL",
+            "value": "ON"
```

Comment:
This doesn't work for me when building on Windows (conda release) in VScode. 
```
[build] C:\Users\dzidm\Downloads\FreeCAD\src\Mod\Fem\App\PreCompiled.h(63): fatal error C1083: Cannot open include file: 'SMDS_PolyhedralVolumeOfNodes.hxx': No such file or directory
```
If I set it to off then `med.h` cannot be found at `file(READ ${meddotH} TMPTXT)` in `SetupSalomeSMESH.cmake`.  

In the end I needed to override `"-DBUILD_FEM=OFF"` as a workaround.

Reply:
It had worked for me previously.  I'll retest.

Reply:
I just re-tested and had no issues with the current configuration.  Furthermore, this is the configuration used in CI: https://github.com/FreeCAD/FreeCAD/blob/main/.github/workflows/sub_buildWindowsConda.yml#L108

It appears the error message is getting caught up in a precompiled header, however these should not be enabled.  By default `FREECAD_USE_PCH` is set to `OFF`: https://github.com/FreeCAD/FreeCAD/blob/main/CMakePresets.json#L82

Reply:
Thank you for checking this. 
I am sorry, I had`FREECAD_USE_PCH` overridden with ON

Reply:
I am uncertain the utility of that setting as it causes build issues for most platforms and I believe it also breaks `ccache` functionality, so it is disabled for all platforms on CI.